sway/config.c

```diff
@@ -657,6 +657,14 @@ char *do_var_replacement(char *str) {
 				continue;
 			}
 		}
+		// Unescape double $ and move on
+		if (find[1] == '$') {
+			size_t length = strlen(find + 1);
+			strncpy(find, find + 1, length);
+			find[length] = '\0';
+			find += 2;
```

Comment:
Since we just erased one `$`, shouldn't this be `+= 1`?

Reply:
Fixed